### ISSUE TITLE
make clean and purge leave test .ipynb files behind

### ISSUE THREAD

**Issue by jfkw** [millejoh/emacs-ipython-notebook#59]
After make purge and/or clean, test files are left behind:
```
% make
(...)
% make purge
rm -f lib/ert/lisp/emacs-lisp/*.elc
rm -f lisp/*.elc
rm -rf env log
% make clean
rm -f lib/ert/lisp/emacs-lisp/*.elc
rm -f lisp/*.elc
% git status
 D "tests/notebook/Islands Tests.ipynb"
?? tests/notebook/Untitled0.ipynb
?? tests/notebook/Untitled1.ipynb
?? tests/notebook/Untitled2.ipynb
?? tests/notebook/Untitled3.ipynb
?? tests/notebook/Untitled4.ipynb
?? tests/notebook/Untitled5.ipynb
?? tests/notebook/Untitled6.ipynb
```

**Comment by millejoh** [comment]
See my comments on #60.

**Comment by jfkw** [comment]
Fixed with 10044ddc9d3f93ac8f423161a600825e26e3d21a.
